code/modules/tgui/modules/crew_monitor.dm

```diff
@@ -23,9 +23,13 @@
 /datum/tgui_module/crew_monitor/tgui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = 0, datum/tgui/master_ui = null, datum/tgui_state/state = GLOB.tgui_default_state)
 	ui = SStgui.try_update_ui(user, src, ui_key, ui, force_open)
 	if(!ui)
-		// The 557 may seem random, but its the perfectsize for margins on the nanomap
-		ui = new(user, src, ui_key, "CrewMonitor", name, 1400, 557, master_ui, state)
+		ui = new(user, src, ui_key, "CrewMonitor", name, 800, 600, master_ui, state)
 		ui.autoupdate = TRUE
```

Comment:
Not really needed, [auto-updating](https://github.com/tgstation/tgstation/blob/8a12f15f409e5f2ca9c4c1a561398fe6cdfde09e/code/modules/tgui/tgui.dm#L25) is already on by default